CharlieBackend.Business/Services/ScheduleServiceFolder/EventsService.cs

```diff
@@ -15,6 +16,7 @@ public class EventsService : IEventsService
     {
         private readonly IUnitOfWork _unitOfWork;
         private readonly IMapper _mapper;
+        private readonly ISchedulesEventsDbEntityVerifier _validator;
```

Comment:
Why don't you set this field in the constructor?

Reply:
Done, thanks